packages/devtools_app/lib/src/utils.dart

```diff
@@ -440,31 +440,33 @@ double safeDivide(num numerator, num denominator, {double ifNotFinite = 0.0}) {
   return ifNotFinite;
 }
 
-/// A simple change notifier.
+/// A change reporter that can be listened to.
+///
+/// Unlike [ChangeNotifier], [Reporter] stores listeners in a set.  This allows
```

Comment:
Add a comment indicating that ChangeNotifier will likely be faster for less than 20 listeners and that this class should only be preferred if you have a very large number of listeners..